packages/react-menu/src/contexts/menuContext.ts

```diff
@@ -10,7 +10,8 @@ const MenuContext = createContext<MenuContextValue>({
   onCheckedValueChange: () => null,
   defaultCheckedValues: {},
   hasMenuContext: false,
-  triggerRef: (null as unknown) as React.MutableRefObject<HTMLElement>,
+  triggerRef: ({} as unknown) as React.MutableRefObject<HTMLElement>,
+  menuPopupRef: ({} as unknown) as React.MutableRefObject<HTMLElement>,
```

Comment:
Let's try to use something closer by a signature:

```suggestion
  triggerRef: React.createRef() as React.MutableRefObject<HTMLElement>,
  menuPopupRef: React.createRef() as React.MutableRefObject<HTMLElement>,
```
or
```suggestion
  triggerRef: { current: null } as React.MutableRefObject<HTMLElement>,
  menuPopupRef: { current: null } as React.MutableRefObject<HTMLElement>,
```